core/threads.go

```diff
@@ -133,8 +133,8 @@ func (t *Textile) RemoveThread(id string) (mh.Multihash, error) {
 }
 
 // Threads lists loaded threads
-func (t *Textile) Threads() []Thread {
-	var threads []Thread
+func (t *Textile) Threads() []*Thread {
```

Comment:
Made this unrelated looking change because of warning about copying a mutex or something

Reply:
very nice